.github/workflows/build.yml

```diff
@@ -0,0 +1,74 @@
+name: Python + Node.js CI
+
+on:
+  # Trigger analysis when pushing in master or pull requests, and when creating
+  # a pull request.
+  push:
+    branches:
+      - main
+  pull_request:
+      types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        # Disabling shallow clone is recommended for improving relevancy of reporting to sonarcloud
+        fetch-depth: 0
+        persist-credentials: false  # To make Github uses personal access token
+
+    # Tests for Python
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Python info
+      shell: bash -l {0}
+      run: |
+        which python3
+        python3 --version
+    - name: Upgrade pip and install dependencies
+      run: |
+        python3 -m pip install --upgrade pip setuptools
+        python3 -m pip install -r requirements.txt
+        python3 -m pip install -r requirements-dev.txt
+    - name: Check style against standards using prospector
+      run: prospector
+    - name: Check import order
+      run: isort --recursive --check-only virtual-coach-server
+    - name: Run unit tests with coverage
+      run: |
+        export PYTHONPATH=$PYTHONPATH:$PWD
+        pytest --cov --cov-report term --cov-report xml --junitxml=xunit-result.xml
+    - name: Correct coverage paths
+      run: sed -i "s+$PWD/++g" coverage.xml
```

Comment:
what does it do? I did not get it, it seems not a replacement

Reply:
I think this removes all $PWD/ in coverage.xml. This is from the [cookiecutter python template](https://github.com/NLeSC/python-template).